src/js/jsx/shared/TextInput.jsx

```diff
@@ -1,26 +1,26 @@
 /*
  * Copyright (c) 2014 Adobe Systems Incorporated. All rights reserved.
- *  
```

Comment:
You gotta turn off your remove-all-trailing whitespace plugin because it makes these diffs really messy. But if somebody wants to go to the trouble to a) in one commit, remove all trailing whitespace from the repo; b) augment one of our grunt tasks to verify the absence of trailing whitespace, optionally c) suggest an editor plugin that will help everyone enforce this automatically, I would merge it!